src/app/project/project-permissions/permissions-table-rows/permissions-table-rows.component.ts

```diff
@@ -21,16 +23,21 @@ export class PermissionsTableRowsComponent implements OnInit, TableComponent {
   @Input() data: TableObject;
 
   public entries: any;
+  public entriesVault: any;
```

Comment:
I saw that `usersVault` below is of type User[]. Are you able to do that here too?

Reply:
Yes I was trying to figure this out... on line 157 I assign the promise return to this.entriesVault. For some reason, the type of the promise return is User (even though it's an array of users). I tried changing the typing down the chain, but then it caused issues in the API file. I will look into it further and try to resolve it with stronger typing.